package.json

```diff
@@ -24,7 +24,7 @@
     "start:dev": "cross-env NODE_ENV=development node ./dist",
     "start:local": "cross-env NODE_ENV=local node ./dist",
     "info": "npm-scripts-info",
-    "build": "trash dist && yarn test && yarn build:main",
+    "build": "trash dist && yarn build:main",
```

Comment:
This makes `build` and `build:dev` identical. What is the reason?

Reply:
Didn't see the `build:dev`...

But it's my opinion that the scripts in the package.json should also follow SRP. So a script named `build` should just build. And a script named `test` should just test. `build` should not both be building and testing etc.
It becomes a hassle when the `build` script in one repository only builds, but in another it both builds and tests, and in a third it builds, tests and lints. Better to keep it simple and say a `build` script only builds.

We should still have a script does all the steps of linting, testing, building etc, but that should be a dedicated script for that and (IMO) `build` is not that dedicated script.

Reply:
But it seems like a good topic to discuss as a team